tremor-script/src/ast/query/raw.rs

```diff
@@ -377,19 +379,27 @@ impl<'script> Upable<'script> for ScriptDefinitionRaw<'script> {
         //
 
         // Handle the content of the script in it's own module
+        let state = self.state.up(helper)?;
         helper.enter_scope();
-        let script = self.script.up_script(helper)?;
+        let mut script = self.script.up_script(helper)?;
+        script.state = state;
         let mid = self.mid.box_with_name(&self.id);
         helper.leave_scope()?;
         // Handle the params in the outside module
         let params = self.params;
         let params = params.up(helper)?;
+        let named = self
```

Comment:
we need to error here, if the `in` port is part of the named scripts and we have, as it would never be executed in the current logic.

Reply:
good call, also prably errors for re-defining the same from section 👍

Reply:
fixed